charts/qdrant/templates/pdb.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if .Values.podDisruptionBudget.enabled }}
+apiVersion: policy/v1
+kind: PodDisruptionBudget
+metadata:
+  name: qdrant
+  labels:
+    {{- include "qdrant.labels" . | nindent 4 }}
+spec:
+  maxUnavailable: {{ .Values.podDisruptionBudget.maxUnavailable }}
```

Comment:
Please also make `minAvailable` configurable and wrap both in an `if`, so that each can be omitted.